tests/test_Dictionary.py

```diff
@@ -28,3 +30,27 @@ class TestGetMany:
     def test_getMany(self):
         d = ee.Dictionary({"foo": 1, "bar": 2}).geetools.getMany(["foo"])
         assert d.getInfo() == [1]
+
+
+class TestToTable:
+    """Test the `toTable` method."""
+
+    def test_to_table_any(self, data_regression):
+        ee_dict = ee.Dictionary({"ADM0_NAME": "Argentina", "ADM0_CODE": "12"})
```

Comment:
Is it still the dictionnary you want to test ? it's a bit weird to have a feature with "ADM0_NAME" and another one "ADM0_CODE". I think the one from the example is more meaningful: 
```python 
d = ee.Dictionary({"foo": 1, "bar": 2})
```

Reply:
I was just tying to mimic FAO's GAUL pattern (level0), but to me is exactly the same as long as it's a `dict` with `str` and/or `int` values.